packages/grid_client/scripts/algorand.ts

```diff
@@ -31,40 +42,44 @@ async function main() {
 
 async function createAccount(account_name: string) {
   try {
-    const account_created = await grid3.algorand.create({
+    const account_created: AlgorandAccountCreateModel = {
```

Comment:
Please make sure to initialize the model in the main function and pass it as a parameter to the other function as you did in all the other scripts.

Reply:
should be fixed in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/1409/commits/cba9251ff5af6781eab04857ecdc1079490b9654